app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ArticlesController < ApplicationController
+	def index
+		@articles = Article.all
+	end
+	def show
+		@article = Article.find(params[:id])
+	end
+	def new
+		@article = Article.new
+	end
+	def create
+		@article = Article.new(article_params)
+		
+		if @article.save
+			redirect_to @article
+		else
+			render 'new'
+		end
+	end
+	private
```

Comment:
When developing in Ruby it's common to use a style guide for the project, to predetermine how files are structured. We tend to use Rubocop to do this automatically (it also integrates well with Atom via a plugin). 

In this case I noticed that `private` does not have a blank line before or after, which we prefer to do.

If you feel like it, I suggest installing the `rubocop` gem and seeing what it suggests in terms of style changes for this project. Note that this is all visual, and does not affect the code. The main reason for having a strict style guide is to reduce confusion and decisions made during development.

I also notice the extra indentation for methods defined after `private`. This is not necessary, since `private` does not define a block (notice how there is no extra `end` statement). `private` is just an indicator to the interpreter that code below this point can not be called from outside of this class.